pkg/apis/app/v2/kieapp_types.go

```diff
@@ -199,10 +201,12 @@ type KieAppObject struct {
 }
 
 type Environment struct {
-	Console     CustomObject   `json:"console,omitempty"`
-	SmartRouter CustomObject   `json:"smartRouter,omitempty"`
-	Servers     []CustomObject `json:"servers,omitempty"`
-	Others      []CustomObject `json:"others,omitempty"`
+	Console          CustomObject   `json:"console,omitempty"`
+	SmartRouter      CustomObject   `json:"smartRouter,omitempty"`
+	Servers          []CustomObject `json:"servers,omitempty"`
+	ProcessMigration CustomObject   `json:"processMigration,omitempty"`
```

Comment:
I assume this means there can only be one PIM object per deployment?

Reply:
Yes, only one PIM object per deployment.